### PR TITLE
Improve contribution guidelines

### PR DESCRIPTION
**Pull request by niveathika**
## Purpose
$subject

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,7 +23,21 @@ For more information on the operations supported by the `sql:Client`, which incl
 
 For example demonstrations of the usage, go to [Ballerina By Examples](https://ballerina.io/swan-lake/learn/by-example/mysql-init-options.html).
 
-## Building from the Source
+## Contributing to Ballerina
+
+As an open source project, Ballerina welcomes contributions from the community.
+
+All Ballerina Standard Library **bug reports/feature requests** are maintained in [ballerina-standard-library](https://github.com/ballerina-platform/ballerina-standard-library) repository.
```
Comment (praneesha) [inline README.md:30]:
```suggestion
All Ballerina Standard Library **bug reports/feature requests** are maintained in the [ballerina-standard-library](https://github.com/ballerina-platform/ballerina-standard-library) repository. 
```

File: README.md [hunk 2]
```diff
@@ -23,7 +23,21 @@ For more information on the operations supported by the `sql:Client`, which incl
...
 For example demonstrations of the usage, go to [Ballerina By Examples](https://ballerina.io/swan-lake/learn/by-example/mysql-init-options.html).
 
-## Building from the Source
+## Contributing to Ballerina
+
+As an open source project, Ballerina welcomes contributions from the community.
+
+All Ballerina Standard Library **bug reports/feature requests** are maintained in [ballerina-standard-library](https://github.com/ballerina-platform/ballerina-standard-library) repository. 
+
+You can also check for [open issues](https://github.com/ballerina-platform/ballerina-standard-library/labels/module%2Fsql) that interest you. We look forward to receiving your contributions. Do not forget to suggest `module/sql` label for issues related to SQL module.
```
Comment (praneesha) [inline README.md:32]:
```suggestion
You can also check for [open issues](https://github.com/ballerina-platform/ballerina-standard-library/labels/module%2Fsql) that interest you. We look forward to receiving your contributions. Do not forget to add the `module/sql` label for issues related to the SQL module.
```

Comment (niveathika) [reply README.md:32]: @praneesha Only committers can add the labels. Others can only suggest in the issues description

**Comment by niveathika** [comment]
Superceeded by https://github.com/ballerina-platform/module-ballerina-sql/pull/37

**Pull request closed without merging**